feature-libs/asm/root/services/asm-enabler.service.ts

```diff
@@ -40,7 +41,8 @@ export class AsmEnablerService {
     protected winRef: WindowRef,
     protected launchDialogService: LaunchDialogService,
     protected lazyModule: LazyModulesService,
-    protected configInitializer: ConfigInitializerService
+    protected configInitializer: ConfigInitializerService,
+    protected featureModulesService: FeatureModulesService
```

Comment:
I wonder how/if we handle this kind of potentially breaking change.  We don't break the existing class, but it might break when users upgrade to 4.0.